### PR TITLE
Make checkboxes checked for default values if no get params entered

### PR DESCRIPTION
**Pull request by prowe12**

### CODE REVIEW

File: cambio/views.py [hunk 1]
```diff
@@ -102,14 +102,14 @@ def index(request: HttpRequest) -> HttpResponse:
...
-    units = [
-        [["carbon_units", "GtC"], ["carbon_units", "atm"]],
-        [["flux_units", "GtC"], ["flux_units", "GtCO2"]],
-        [["temp_units", "C"], ["temp_units", "K"], ["temp_units", "F"]],
-        [],
-        [],
-    ]
-    # always plotted: "albedo", "pH"
+
+    # units = [
```
Comment (rowedaniel) [inline cambio/views.py]: delete the commented-out code?

File: cambio/views.py [hunk 2]
```diff
@@ -120,6 +120,70 @@ def index(request: HttpRequest) -> HttpResponse:
     flux_vars = get_scenario_vars(request, flux_vars_to_plot)
     temp_vars = get_scenario_vars(request, temp_vars_to_plot)
 
+    print("scenario units")
```
Comment (rowedaniel) [inline cambio/views.py]: delete print statements?

File: cambio/views.py [hunk 3]
```diff
@@ -120,6 +120,70 @@ def index(request: HttpRequest) -> HttpResponse:
...
+        "albedo": {
+            "plot": [],
+            "albedo": ["albedo"],
+            "units": [],
+            "selected_vars": [],
+            "selected_units": [],
+        },
+    }
+
+    print("")
```
Comment (rowedaniel) [inline cambio/views.py]: more print statements

File: cambio/views.py [hunk 4]
```diff
@@ -120,6 +120,70 @@ def index(request: HttpRequest) -> HttpResponse:
...
+    print("")
+    print("scenario_units")
+    print(scenario_units)
+    print("")
+    print("")
+
+    # Replace the default plotting variables with the get params
+    if any(carbon_vars):
```
Comment (rowedaniel) [inline cambio/views.py:148]: pretty sure any([0,0,0]) returns false -- is this intended behavior, or do you want if len(carbon_vars) > 0?

File: cambio/views.py (hunk 1 above)
Comment (prowe12) [reply cambio/views.py]: Will do.

File: cambio/views.py (hunk 3 above)
Comment (prowe12) [reply cambio/views.py]: Will remove.

File: cambio/views.py (hunk 4 above)
Comment (prowe12) [reply cambio/views.py:148]: It is intended behavior, but will be removed at the next refactor.

File: cambio/views.py (hunk 2 above)
Comment (prowe12) [reply cambio/views.py]: Done

**Pull request merged**